test/test_suite.py

```diff
@@ -68,3 +73,21 @@ def test_percent_sign_round_trip(self):
                 ),
                 "some %% other value",
             )
+
+
+class ExceptionTest(_ExceptionTest):
+    @config.requirements.duplicate_key_raises_integrity_error
+    def test_integrity_error(self):
+        with config.db.connect() as conn:
+            conn.connection.connection.autocommit = True
+            trans = conn.begin()
+            conn.execute(self.tables.manual_pk.insert(), {"id": 1, "data": "d1"})
+
+            assert_raises(
+                exc.IntegrityError,
+                conn.execute,
+                self.tables.manual_pk.insert(),
+                {"id": 1, "data": "d1"},
+            )
+
+            trans.rollback()
```

Comment:
This changes the behavior of the test. It was working fine before the PR 223, more than that, there are at least 3 other tests, which are failing because of 223 PR. Thus, I assume it should be fixed in python-spanner and insert helpers/parsers, not by patching every failing test.